crates/api/src/op_store.rs

```diff
@@ -91,6 +91,22 @@ pub trait OpStore: 'static + Send + Sync + std::fmt::Debug {
         op_ids: Vec<OpId>,
     ) -> BoxFuture<'_, K2Result<Vec<MetaOp>>>;
 
+    /// Retrieve a size-bounded list of op ids that have been stored since `start`.
+    ///
+    /// The `limit_bytes` applies to the size of the op data, not the size of the op ids.
+    ///
+    /// # Returns
+    ///
+    /// As many op ids as can be returned within the `limit_bytes` limit.
+    ///
+    /// If the limit is applied, then the timestamp of the last op id returned is returned.
+    /// Otherwise, the timestamp for when this operation started is returned.
+    fn retrieve_op_ids_bounded(
```

Comment:
This is new and is intended to be used for either incremental get or with the DHT diffing when that's integrated. This will need modifying to respect the common arc set at some point which means moving that from the DHT model to the API. Think that's better left for another PR while we're doing full arc only for now